google/cloud/documentai_toolbox/utilities/docai_utilities.py

```diff
@@ -43,6 +44,9 @@ def get_bounding_box(
         )
         for vertex in bounding_poly.normalized_vertices
     ]
+    if not vertices:
+        return 0, 0, 0, 0
+
     top, left = vertices[0]
     bottom, right = vertices[2]
```

Comment:
Before this PR, this `vertices[0]` could result in an `IndexError`, also it is still unclear what happens to `vertices[1]`.  I would suggest eventually restructure and improve the code here (not blocking this PR).  We also seem to be assuming that `bounding_poly` is always a bounding box (as opposed to more general polygons) - do we know that the service always returns boxes?

Additionally, if the utility function needs only `bounding_poly.normalized_vertices`, it should receive just a list of normalized vertices and not the whole `bounding_poly`.